datagenerator/src/main/resources/rythm/common_header.rythm

```diff
@@ -1 +0,0 @@
-// DO NOT EDIT - THIS FILE IS GENERATED BY restli-csharp
```

Comment:
This file can take one argument `CSharpType`, then have one line
// DO NOT EDIT - THIS FILE IS GENERATED BY restli-csharp@cSharpType.restli_generatedFrom()
Then all other rythms just share this one common_header, in the future, if we need to change header, we only need to change this one file.